CliWrap/StandardBufferHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace CliWrap
+{
+    /// <summary>
+    /// The standard implementation of <see cref="IStandardBufferHandler"/>
+    /// </summary>
+    public class StandardBufferHandler : IStandardBufferHandler
+    {
+        private readonly Action<string> _standardOutputHandler;
+        private readonly Action<string> _standardErrorHandler;
+
+        /// <summary>
+        /// Sets up a standard buffer handler using two callbacks. Both callbacks are optional.
+        /// </summary>
+        /// <param name="standardOutputHandler">The callback to use when it gets written to the standard output</param>
+        /// <param name="standardErrorHandler">The callback to use when it gets written to the standard error</param>
+        public StandardBufferHandler(Action<string> standardOutputHandler = null, Action<string> standardErrorHandler = null)
+        {
+            _standardOutputHandler = standardOutputHandler;
+            _standardErrorHandler = standardErrorHandler;
+        }
+
+        /// <summary>
+        /// Calls the callback
```

Comment:
You can just use `/// <inheritdoc />` on the following two methods to have them inherit comments from the interface